Rakefile

```diff
@@ -24,7 +24,12 @@ namespace :docker do
 
   desc "Build docs using docker"
   task :build do
-    exec 'docker run --rm -i -t -v "${PWD}:/doc" -u "$(id -u):$(id -g)" ohiosupercomputer/docker-sphinx make html'
+    exec 'docker run --rm -i -t -v "${PWD}:/doc" -u "$(id -u):$(id -g)" --security-opt label=disable ohiosupercomputer/docker-sphinx make html'
+  end
+  
+  desc "Build drawio images"
+  task :build_images do
+    exec 'docker run --rm -i -t -v "${PWD}:/doc" -u "$(id -u):$(id -g)" --security-opt label=disable ohiosupercomputer/docker-sphinx node ../drawio-batch/drawio-batch.js source/architecture/ood_container_view.drawio source/images/ood_container_view.png && node ../drawio-batch/drawio-batch.js source/architecture/ood_system_view.drawio source/images/ood_system_view.png'
```

Comment:
Thoughts:

- When I tested I didn't need the `--security-opt label=disable` to run, so that may be specific to your set up.
- Instead of running `node ../drawio-batch.js` you can just run `drawio-batch` because it is installed to `/usr/bin/drawio-batch` which is in `$PATH`.
- This should use a `find` command (or similar using Python or NodeJS) that processes found `*.drawio` files instead of processing a hard-coded list.
- I would expect that a single command would both generate the images and then generate the HTML.
- Whatever the solution turns out to be will need to also be implemented for the Makefile since that is what TravisCI is using. That can either be in this PR or a separate PR when we have worked out the issues with this one.

Reply:
I just comitted and pushed the changes requested.